elections/uk/lib.py

```diff
@@ -24,38 +26,43 @@ def shorten_post_label(post_label):
     'party_ec_id',
 ]
 
-def get_extra_csv_values(person, election):
-    theyworkforyou_url = ''
+def get_extra_csv_values(person, election, post):
+    gss_code = ''
```

Comment:
This is in `elections/uk` so at least we don't have to worry about the other countries. It would be good to add other columns for the other area types that are used in candidates.dc, but for the moment this is just making sure the output is the same as before, just faster.